src/OpenTelemetry.Shims.OpenTracing/SpanContextShim.cs

```diff
@@ -28,7 +28,7 @@ public SpanContextShim(in Trace.SpanContext spanContext, IEnumerable<KeyValuePai
         {
             if (!spanContext.IsValid)
             {
-                throw new ArgumentException(nameof(spanContext));
+                throw new ArgumentException($"{nameof(spanContext)} must be valid.");
```

Comment:
It is a bit weird to see a message like "something must be valid".

Reply:
Hum. Do u have something in ur mind?